bundles/org.openhab.binding.amazonechocontrol/README.md

```diff
@@ -180,6 +182,10 @@ Sample for the Thing echo1 only. But it will work in the same way for the other
 Take a look in the channel description above to know, which channels are supported by your thing type.
 
 ```
+// Account
+String Echo_Living_Room_SendMessage            "SendMessage"                           {channel="amazonechocontrol:account:account1:sendMessage"}
+String Echo_Living_Room_AccountAnnouncement    "AccountAnnouncement"                   {channel="amazonechocontrol:account:account1:accountAnnouncement"}
```

Comment:
Shouldn't the cannel be named `sendAnnouncement` to keep it consistent with the `sendMessage`?